### PR TITLE
Update release list in the main README

### PR DESCRIPTION
**Pull request by sebbader**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ https://licensebuttons.net/l/by/4.0/88x31.png
 https://creativecommons.org/licenses/by/4.0/
 )
 
-This repository contains the specifications of the Asset Administration Shell, including the schemas of the serializations, the rules applied, how the specification is mapped into the serializations, and examples of how to use the schemas.
+This repository contains specifications of the Asset Administration Shell, including the normative schemas of the serializations, the rules applied to create them, how the specification is mapped into the serializations, and examples of how to use the schemas.
 
 ## Schemas
 
-We provide schemas of the Asset Administration Shell for JSON, RDF and XML. 
+The schemas of the Asset Administration Shell for JSON, RDF and XML are provided in the respective directories.
```
Comment (BirgitBoss) [inline README.md:18]: Additionally the xmi file of the UML modeling is provided.
What about YAML?

File: README.md [hunk 2]
```diff
@@ -36,16 +36,22 @@ The XML schema, mapping rules and examples are available at [schemas/xml/](schem
...
+* [3.0.4.RC01](https://github.com/admin-shell-io/aas-specs/releases/tag/v3.0.4.RC01) is the final release for the `V3.0RC01` version of the AAS metamodel, containing the normative schemas for the published document "Part 1: Details of the Asset Administration Shell - The exchange of information between partners in the value chain of Industrie 4.0 - **Version 3.0RC01**". *No additional bugfix or patch releases for the metamodel version `V3.0RC01` shall be expected.*
+* [3.0.3.RC01](https://github.com/admin-shell-io/aas-specs/releases/tag/v3.0.3.RC01) is a pre-release of the schemas for the published document "Part 1: Details of the Asset Administration Shell - The exchange of information between partners in the value chain of Industrie 4.0 - Version 3.0RC01" that solves several typos and bugs of `3.0.2.RC01`
```
Comment (BirgitBoss) [inline README.md:51]: evtl. reichen auch nur die 3.0.4.RC01 und 3.0.5.RC02 da die anderen ja nicht mehr verwendet werden sollen, aber falsch ist es natürlich nicht

I miss chapter on Versioning schema:

We use the following versioning scheme:
<major version>.<minor version>.<bugfix version>.RC<number>

<major version> and <minor version> as well as RC<number> are identical to the UML specification document. Patches on the schemas are denoted by the <bugfix version>. 
RC stands for "Release Candidate", there can be more than one release candidate per UML specification version. If the UML specification document itself has a bugfix version then this is denotes by
<major version>.<minor version>.<bugfix version>.RC<number>-patch<Bugfix version of UML specification>

File: README.md [hunk 3]
```diff
@@ -61,5 +67,5 @@ If you want to contribute, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 This work is licensed under a [Creative Commons Attribution 4.0 International License](
 https://creativecommons.org/licenses/by/4.0/). The implementation uses the concepts of the
```
Comment (BirgitBoss) [inline README.md:69]: instead of "uses the concepts" perhaps : "is based on the specifications of"

File: README.md [hunk 4]
```diff
@@ -61,5 +67,5 @@ If you want to contribute, see [CONTRIBUTING.md](CONTRIBUTING.md).
...
-document "Details of the Asset Administration Shell" published on www.plattform-i40.de
-which is licensed under Creative Commons CC BY 4.0.
+document "Details of the Asset Administration Shell" published on https://www.plattform-i40.de
```
Comment (BirgitBoss) [inline README.md:70]: and idtwin.org

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
minor changes + chapter on versioning schema (please review whether my proposal is adequate and correct)

File: README.md (hunk 1 above)
Comment (sebbader) [reply README.md]: I added statements for YAML and XMI

File: README.md (hunk 3 above)
Comment (sebbader) [reply README.md]: done

**Review by BirgitBoss — APPROVED** [review]

**Pull request merged**